### PR TITLE
Add STS to describe cluster output

### PR DESCRIPTION
**Pull request by fahlmant**
Shows STS true or false based on the existence of STS information for the cluster.

### CODE REVIEW

File: pkg/cluster/describe.go [hunk 1]
```diff
@@ -130,6 +130,11 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
 		privateLinkEnabled = cluster.AWS().PrivateLink()
 	}
 
+	stsEnabled := false
+	if cluster.AWS().STS() != nil {
```
Comment (pvasant) [inline pkg/cluster/describe.go:134]: Do we need nil validation for 

 if cluster.AWS()!=nil  &&  cluster.AWS().STS() != nil

Comment (fahlmant) [reply pkg/cluster/describe.go:134]: Good point, let me put it in the same block as the privateLink check above

Comment (vkareh) [reply pkg/cluster/describe.go]: No, this should be fine, as the types are initialized. However, this also means that checking for `STS != nil` is not sufficient, you'll need something like:
```go
if cluster.AWS().STS().RoleARN() != "" {
  stsEnabled = true
}
```

File: pkg/cluster/describe.go [hunk 2]
```diff
@@ -126,8 +126,12 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
 	}
 
 	privateLinkEnabled := false
+	stsEnabled := false
 	if cluster.CloudProvider().ID() == ProviderAWS && cluster.AWS() != nil {
 		privateLinkEnabled = cluster.AWS().PrivateLink()
+		if cluster.AWS().STS() != nil {
```
Comment (vkareh) [inline pkg/cluster/describe.go:132]: Use `cluster.AWS().STS().RoleARN() != ""`

Comment (fahlmant) [reply pkg/cluster/describe.go:132]: Makes sense, will do

**Pull request merged**